src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -1,9 +1,208 @@
-import { Text } from "@artsy/palette"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Column,
+  Flex,
+  GridColumns,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { Formik } from "formik"
 
 export const PreferencesApp: React.FC = () => {
   return (
     <>
-      <Text variant="xl">Preferences App</Text>
+      <Text variant="xl" mt={4} mb={12}>
+        Preferences Center
+      </Text>
+
+      <Separator mt={2} />
+
+      <GridColumns>
+          <Column
+            span={10}
+            mt={2}
+          >
+            <Flex p={2}>
+              <Box>
+                <Text variant="md">Recommended By Artsy</Text>
+                <Text variant="sm" color="black60">
+                  Artworks, shows, fairs, auctions, and collections we think you'll
+                  love
+                </Text>
+              </Box>
+            </Flex>
+
+            <Flex p={2}>
+              <Box>
+                <Text variant="md">Art World Insights</Text>
+                <Text variant="sm" color="black60">
+                  Market stories, artist profiles, exhibition reviews, and more art
+                  world insights
+                </Text>
+              </Box>
+            </Flex>
+
+            <Flex p={2}>
+              <Box>
+                <Text variant="md">Product Updates</Text>
+                <Text variant="sm" color="black60">
+                  Announcements of new features on Artsy.net and the mobile app
+                </Text>
+              </Box>
+            </Flex>
+
+            <Flex p={2}>
+              <Box>
+                <Text variant="md">Guidance on Collecting</Text>
+                <Text variant="sm" color="black60">
+                  Expert advice on buying and selling art, directly from an Artsy
+                  specialist
+                </Text>
+              </Box>
+            </Flex>
+
+            <Flex p={2}>
+              <Box>
+                <Text variant="md">Custom Alerts</Text>
+                <Text variant="sm" color="black60">
+                  A round up of updates on your favorite artists, chosen by you
+                </Text>
+              </Box>
+            </Flex>
+
+            <Separator mt={2}/>
```

Comment:
@dzucconi I am having issues utilizing the `<Join>` element to apply a `<Separator>` between the email options and the unsubscribe from all options. How would I successfully apply them between the `<Column>` elements?